### DISCUSSION TITLE
Add "little image" when posting a link of the webpage.

### DISCUSSION THREAD

**Discussion by g-pacheco** [athul/archie#64] [General]
Hello all, 

I am sorry -- I do not know the proper terms here. But I was wondering how I could make the links of my webpage _preview_ their content when they are posted on something like Twitter. My links (https://g-pacheco.github.io/) currently do not do that :( 
However, I saw that the Archie demo`s links do that. So I wonder what I could be missing.

<img width="1015" alt="Screenshot 2022-11-04 at 17 07 59" src="https://user-images.githubusercontent.com/37849281/200025161-480e5da3-ab9e-40bd-9d13-b1a31e7a3863.png">
<img width="1025" alt="Screenshot 2022-11-04 at 17 07 42" src="https://user-images.githubusercontent.com/37849281/200025110-3f0bd82b-c78f-48ef-8ce2-c364e89a579a.png">

Many thanks in advance, George.

**Comment by athul** [comment]
Hi This is called Meta Images. You can use meta tags in HTML to do the same. Sadly this is not built into archie, you might want to hardcode the same
